### PR TITLE
rarun2 man page + clearenv fcn

### PR DESCRIPTION
**Pull request by grandemk**
fix unsetenv wrong description
create clearenv keyword to erase all the env var
add some example for environment variable
fix r2 use of rarun2 example

### CODE REVIEW

File: libr/util/sys.c [hunk 1]
```diff
@@ -187,6 +190,22 @@ R_API int r_sys_usleep(int usecs) {
 #endif
 }
 
+R_API int r_sys_clearenv(void)
```
Comment (ret2libc) [inline libr/util/sys.c]: Seems like the #if branch doesn't return a value when ```environ != NULL```

Comment (grandemk) [inline libr/util/sys.c]: my bad, forgot to push the last commit to my rep, will fix.

**Comment by ret2libc** [comment]
Thanks @grandemk for the PR! Seems to work for me.
It would be cool to have clearenv option also in r_run_help (commented with #) imho.

Also, please have a look at https://github.com/radare/radare2/blob/master/doc/syntax.md. In particular, check the indentation of your patch and the positioning of braces when you define a new function.

**Comment by grandemk** [comment]
Hadn't seen it, why tab for spacing ... Well it's good to have a coding style. I think this should be a CODINGSTYLE.md at the root of the project.

**Comment by ret2libc** [comment]
Maybe you are right, thanks for the suggestion! Btw, the patch looks good to me now.
Let's wait for @radare

**Comment by radare** [comment]
Lgtm too, just squash and rebase to avoid too much git log noise.

Thanks!

**Comment by grandemk** [comment]
done

**Comment by ret2libc** [comment]
Thanks a lot! I've merged it. I just needed to add a '\n' in the r_run_help change ;)

**Pull request closed without merging**